### PR TITLE
Fix README: fix the wrong word and delete line number description of the README, which belong to Python example `data_transmission` 

### PR DESCRIPTION
**Pull request by kerbalwzy**
@karthikravis
There have a wrong word in two README file, the `clien.py` should be `client.py`, and the line number description of the target functions and method also have wrong after add copyright headers. 
Considering that any modification to the code will cause a line number change, it would be better to delete the line number description here.
In fact, this example was also PR by myself.

### CODE REVIEW

File: examples/python/data_transmission/README.cn.md [hunk 1]
```diff
@@ -6,31 +6,31 @@
...
 
-  `server.py - line:17 - SimpleMethod`
+  `server.py : SimpleMethod`
 
 - #### 客户端流模式
 
   在一次调用中, 客户端可以多次向服务器传输数据, 但是服务器只能返回一次响应
 
-  `clien.py - line:27 - client_streaming_method `
+  `client.py : client_streaming_method `
```
Comment (kerbalwzy) [inline examples/python/data_transmission/README.cn.md]: fix wrong word

File: examples/python/data_transmission/README.cn.md [hunk 2]
```diff
@@ -6,31 +6,31 @@
 
   在一次调用中, 客户端只能向服务器传输一次请求数据, 服务器也只能返回一次响应
 
-  `client.py - line:14 - simple_method`
+  `client.py : simple_method`
```
Comment (kerbalwzy) [inline examples/python/data_transmission/README.cn.md]: remove all line number descriptions

File: examples/python/data_transmission/README.en.md [hunk 3]
```diff
@@ -6,32 +6,32 @@ Four ways of data transmission when gRPC is used in Python.  [Offical Guide](<ht
 
   In a single call, the client can only send request once, and the server can only respond once.
 
-  `client.py - line:14 - simple_method`
+  `client.py : simple_method`
```
Comment (kerbalwzy) [inline examples/python/data_transmission/README.en.md]: remove all line number descriptions also.

File: examples/python/data_transmission/README.en.md [hunk 4]
```diff
@@ -6,32 +6,32 @@ Four ways of data transmission when gRPC is used in Python.  [Offical Guide](<ht
...
 
-  `server.py - line:17 - SimpleMethod`
+  `server.py : SimpleMethod`
 
 - #### stream-unary
 
   In a single call, the client can transfer data to the server an arbitrary number of times, but the server can only return a response once.
 
-  `clien.py - line:27 - client_streaming_method`
+  `client.py : client_streaming_method`
```
Comment (kerbalwzy) [inline examples/python/data_transmission/README.en.md]: fix the wrong word also

File: examples/python/data_transmission/README.cn.md (hunk 2 above)
Comment (gnossen) [inline examples/python/data_transmission/README.cn.md]: Any reason for it not to be `client.py: simple_method`? [Colons generally have no space on the left, but a space on the right](https://en.wikipedia.org/wiki/Colon_(punctuation)#Usage_in_English).

File: examples/python/data_transmission/README.cn.md [hunk 5]
```diff
@@ -6,31 +6,31 @@
...
 
-  `server.py - line:17 - SimpleMethod`
+  `server.py : SimpleMethod`
```
Comment (lidizheng) [inline examples/python/data_transmission/README.cn.md]: Can we either change the formating to:
```Markdown
* `client.py: simple_method`
* `server.py: SimpleMethod`
```
or
```Markdown
* `client.simple_method`
* `server.DemoServer.SimpleMethod`
```

Apply to all other code reference.

Comment (kerbalwzy) [reply examples/python/data_transmission/README.cn.md]: I chose the first style.

**Comment by kerbalwzy** [comment]
@lidizheng 
Having one checking was failed here, I don't know why. 
I had saw the log information, but still not have any idea. Can you tell me how to handle the problem ?:
image

**Comment by lidizheng** [comment]
@kerbalwzy The tests are looking good to me. Next time please save the link to test result.

**Pull request merged**